Sources_v3/StreamChatUI/ChatChannel/ChatMessageContentView.swift

```diff
@@ -10,6 +10,13 @@ open class ChatMessageContentView<ExtraData: UIExtraDataTypes>: View, UIConfigPr
         didSet { updateContent() }
     }
 
+    public var onThreadTap: (_ChatMessageGroupPart<ExtraData>?) -> Void = { _ in } {
+        didSet {
+            guard superview != nil else { return }
+            threadView.onTap = onThreadTap
```

Comment:
nit: You can maybe just use `updateContentIfNeeded` so we have `threadView.onTap = onThreadTap` on a single place?